salt/states/kubernetes.py

```diff
@@ -4,7 +4,7 @@
 ==========================================
 
```

Comment:
Could this have been solved using `absolute_imports` here?

Reply:
No, because we add salt/modules to sys.path in the loader.

Reply:
https://github.com/saltstack/salt/blob/2018.3/salt/loader.py#L1453